### ISSUE TITLE
RMarkdown required but not listed in dependencies

### ISSUE THREAD

**Issue by dhvalden** [annajenul/UBayFS#22]
Under Fedora Linux 37, R version 4.2.2 (2022-10-31), suggested package installation `remotes::install_github("annajenul/UBayFS", build_manual = TRUE, build_vignettes = TRUE)`.

Error message:
```
Running `R CMD build`...
* checking for file ‘/tmp/Rtmp5V5S0y/remotes118345827dd25/annajenul-UBayFS-c493962/DESCRIPTION’ ... OK
* preparing ‘UBayFS’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘BFS_UBayFS.Rmd’ using rmarkdown
Error: processing vignette 'BFS_UBayFS.Rmd' failed with diagnostics:
Pandoc is required to build R Markdown vignettes but not available. Please make sure it is installed.
--- failed re-building ‘BFS_UBayFS.Rmd’
```

**Comment by dhvalden** [comment]
Just a note, I can easily bypass the issue by just not building the vignettes, so my suggested solution would just to change the suggested installation to not build vignettes by default.

**Comment by dhvalden** [comment]
ok! i just noticed this got solved with commit 49b5245. Closing as solved
